code/src/java/plugin/converter/FavClassConvertPlugin.java

```diff
@@ -54,9 +55,15 @@ public String process(TokenProcessEvent tpe)
 		}
 
 		String choices = value.substring(7);
-		if (isEmpty(choices) || hasIllegalSeparator('|', choices))
+		ParseResult pr = checkNonEmpty(choices);
+		if (!pr.passed())
 		{
-			return "Empty Token";
+			return "Empty";
```

Comment:
extract Magic Const

Reply:
This will require discussion - I think we have different definitions of "magic"

Reply:
Is it used more than once?